docs-site/docs/docs/text-cells.md

```diff
@@ -12,17 +15,39 @@ The [Text Toolbar](toolbar.md#text-cells) contains standard Rich Text Editing el
 
 It also contains a few non-Standard options.
 
-### Code and Links
+#### Code and Links
 
 The **code** button tags the currently-selected text as `code`, formatting it as monospaced. 
 
 The **link** button tags the currently-selected text as a link. Clinking on a link will reveal the URL in a tooltip, 
 allowing you to navigate to the website. 
 
-![Code and Link](images/text-cell-code-link.gif)
+![Code and Link](images/text-cell-code-link.gif){: .centered-image }
 
-### LaTeX Editor
+#### LaTeX Editor
 
 You can also input `LaTeX` into your text cells, using the button that looks like a Sigma. 
 
-![LaTeX Editor](images/text-cell-latex.gif)
\ No newline at end of file
+![LaTeX Editor](images/text-cell-latex.gif){: .centered-image }
+
+### Markdown Editing 
+The Markdown Editor allows you to edit text in traditional `atx` style markdown in a code editor and see it rendered neatly
+as traditional compiled markdown.  
+
+To enable markdown editing in place of traditional Rich Text Editing, you must set the `markdown` setting to `true` in 
+the [About menu](about-menu.md). 
+
+![Markdown Editor](images/markdown-editor.gif) {: .centered-image }
+
+#### Supported Markdown Syntax 
```

Comment:
Where did this list come from? AFAIK, we support whatever markdown-it supports (including the TeX plugin we've added)

Reply:
Hmmm, I think this maybe came from the code that we're currently using to convert HTML back to markdown in the RTE? But that list doesn't apply to raw markdown.

Reply:
I think the explicit list I gave is kinda iffy (will remove it). I took it directly from `markdown-it`'s sample of what syntax they support - I think just linking the example notebook (which uses the exact example from their website) would suffice?

Reply:
Actually this section probably isn't even necessary? TBH we support regular markdown which makes it fairly self-explanatory.

Reply:
I agree with removing that bit. Looks good now.